dev/archery/archery/docker/cli.py

```diff
@@ -47,18 +47,24 @@ def _execute(self, *args, **kwargs):
               help="Specify Arrow source directory.")
 @click.option('--dry-run/--execute', default=False,
               help="Display the docker commands instead of executing them.")
+@click.option('--using-legacy-docker-compose', default=False, is_flag=True,
+              envvar='ARCHERY_USE_LEGACY_DOCKER_COMPOSE',
+              help="Use legacy docker-compose utility instead of the built-in "
+                   "`docker compose` subcommand. This may be necessary if the "
+                   "Docker client is too old for some options.")
 @click.option('--using-docker-cli', default=False, is_flag=True,
               envvar='ARCHERY_USE_DOCKER_CLI',
               help="Use docker CLI directly for building instead of calling "
-                   "docker-compose. This may help to reuse cached layers.")
+                   "`docker compose`. This may help to reuse cached layers.")
```

Comment:
Can we remove this option?

Reply:
Well, apparently it's necessary for Windows wheels:
https://github.com/ursacomputing/crossbow/actions/runs/10269082430/job/28413695620

Reply:
Update: it's not if I ensure that buildkit is disabled.